agent/agent.go

```diff
@@ -57,6 +57,7 @@ type orbAgent struct {
 type GroupInfo struct {
 	Name      string
 	ChannelID string
+	policyIDs []string
```

Comment:
It will be easier to just make this a map keyed by policy ID (with a boolean value of true) to avoid the "contains()" func and the array logic